### PR TITLE
20162 Fixed logic for showing (vs disabling) limited restoration staff actions

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* bcgov/entity#20162

*Description of changes:*
- app version = 7.1.0
- show limited restoration staff actions according to allowable actions
- updated unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.0.49",
+  "version": "7.1.0",
```
Comment (severinbeauvais) [inline package.json:3]: I'm going to say that all changes up to 7.0.49 were "patches" to whatever we were working on before, and now this ticket a new topic (restoration stuff).

Lmk if you think I should increment to v7.0.50 instead.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-632-3g4yss0c.web.app

**Comment by severinbeauvais** [comment]
Previously:

image

Now:

image

**Review by jamespaologarcia — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (ketaki-deodhar) [reply package.json:3]: I think 7.1.0 is better

**Review by ketaki-deodhar — APPROVED** [review]
Looks good!

**Pull request merged**